CHANGES.rst

```diff
@@ -33,6 +35,54 @@ Changed
 - Packets blocked due to rate limits will be now dropped instead of being
   rejected by default. [gaudenz]
 
+- The data format of the firewall rules has been redesigned. Rules can now be
+  defined as nested YAML dictionaries, existing default or dependent rules can
+  be easily modified through the Ansible inventory, multiple firewall rules can
+  be included in one configuration file. The role is compatible with the old,
+  list-based data format, however dictionary-based format should be preferred.
```

Comment:
Please update this entry. You only need to track changes across released versions 😉 So if a change has never been released there is no need to add it here. It is tracked in git :)